apix-impl/src/integration-test/java/eu/xenit/apix/rest/v2/tests/AllNodeInfoTest.java

```diff
@@ -79,6 +80,49 @@ public void testGetAllNodeInfoOfMultipleNodes() throws IOException, JSONExceptio
         }
     }
 
+    @Test
+    public void testGetAllNodeInfoWithNoNodesListed() throws IOException {
+        String jsonString = json("{}");
+
+        final CloseableHttpClient httpclient = HttpClients.createDefault();
+        final String url = makeAlfrescoBaseurl("admin", "admin") + "/apix/v2/nodes/nodeInfo";
+        logger.error("url: " + url);
+        HttpPost httppost = new HttpPost(url);
+        httppost.setEntity(new StringEntity(jsonString));
+
+        try (CloseableHttpResponse response = httpclient.execute(httppost)) {
+            assertEquals(400, response.getStatusLine().getStatusCode());
+        }
+    }
+
```

Comment:
Maybe add a test for some non-existing and no-permissions nodes together with some real nodes and assert the amount of nodeInfo's returned